internal/commands/commands.go

```diff
@@ -177,4 +178,19 @@ var (
 			},
 		},
 	}
+
+	AccessList = cli.CommandDefinition{
+		CommandMeta: cli.CommandMeta{
+			Use:         "accessList",
```

Comment:
I do believe this was discussed and we want to stay consistent with Atlas naming conventions for this one and stick with `accessList` as the default and `accesslist` as the alias. Here's the [discussion](https://docs.google.com/document/d/1foVVydfAxTsA4rVPLz3x4YDlHcnXBgeDKgD3a2RPBtU/edit?disco=AAAAMLbjyu4) behind it.

Reply:
I'd love to hear the rationale that was used "per project check in meeting", b/c it seemed like that whole discussion up to that point was agreeing with what I suggested here.

you can put the blame on me, but can we just have one last thread on slack about the naming here?  b/c 2 days after that discussion I know we had zoom call (christine sumedha and i) clearing up some confusion about the naming of these commands

Reply:
Just had a discussion with Sumedha, seems like we're going to go with `accesslist` as the default and `access-list` as an alias for now!

Reply:
Since this is a security based command, we should stay consistent with atlas (`accessList`) as the default. I'm fine with others as aliases. The last thing we want to do is add confusion or cognitive overhead to a security control.

best case scenario here would be to have mcli change their default, but that's a long shot.